app/services/proofing/result.rb

```diff
@@ -8,13 +8,15 @@ def initialize(
       context: {},
       exception: nil,
       transaction_id: nil,
-      reference: nil
+      reference: nil,
+      response_body: nil
     )
       @errors = errors
       @context = context
       @exception = exception
       @transaction_id = transaction_id
       @reference = reference
+      @response_body = response_body
```

Comment:
regretting I named this response_body since it's a hash

Reply:
It's just code, we can always rename it